tests/tests/rap_noah_sfcdiff_unified_ugwp

```diff
@@ -0,0 +1,95 @@
+##########################################################################################
```

Comment:
Not in history of develop. Does this go with that change to the xml?

Currently in develop:
rap_noah_debug
rap_noah_sfcdiff_cires_ugwp_debug